package.json

```diff
@@ -318,7 +312,9 @@
     "dev": "vite --host",
     "build": "cross-env NODE_OPTIONS=--max-old-space-size=8192 vite build",
     "preview": "vite preview",
-    "build-storybook": "cross-env NODE_OPTIONS=--max-old-space-size=6144 storybook build -o build/storybook",
+    "sentry:inject": "sentry-cli sourcemaps inject build/dashboard",
+    "sentry:upload": "sentry-cli sourcemaps upload build/dashboard",
+    "sentry:clean": "node scripts/removeSourcemaps.js",
```

Comment:
Unrelated changes, are those accidental?

Reply:
Good catch @NyanKiyoshi, we should delete them

Reply:
Removed :bo